nexus/reconfigurator/planning/src/blueprint_editor/sled_editor.rs

```diff
@@ -143,17 +416,21 @@ impl SledEditor {
         }
     }
 
-    pub fn set_state(&mut self, new_state: SledState) {
-        self.state = new_state;
-    }
-
     pub fn disks(
         &self,
         filter: DiskFilter,
     ) -> impl Iterator<Item = &BlueprintPhysicalDiskConfig> {
         self.disks.disks(filter)
     }
 
+    #[allow(dead_code)] // currently only used by tests; this will change soon
```

Comment:
Should we make this `#[cfg(test)]` for now instead?

Reply:
Ah, this is used by the commented-out block of checks you suggested moving to an issue :sweat_smile: I'll do the issue thing and then remove this method.

Reply:
Removed in 79535ec18